lib/KalturaBase.js

```diff
@@ -12,6 +12,9 @@ var kaltura = {
 require('./utils/KalturaUtils');
 require('./utils/KalturaConfig');
 require('./utils/KalturaCache');
+require('./utils/KalturaCacheManager');
+require('./utils/KalturaCouchbaseConnector');
```

Comment:
I think only the cache manager needs the connector - the others should not be aware that they exist